packages/apps/docs/src/scripts/utils.ts

```diff
@@ -36,303 +36,8 @@ export interface IImportReadMeItem {
  */
 export const importReadMes: IImportReadMeItem[] = [
```

Comment:
why are we keeping this empty array with commented codes?

Reply:
good point